src/main/resources/avro/variantmethods.avdl

```diff
@@ -148,68 +148,3 @@ org.ga4gh.models.Variant getVariant(
   The ID of the `Variant`.
   */
   string id) throws GAException;
-
-/******************  /callsets/search  *********************/
```

Comment:
How do I get a list of all the columns in a variantset without some equivalent of this method?